.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Oh no! A bug!
+title: ''
+labels: ''
+assignees: mbax
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Stacktrace**
+Paste the stacktrace you get here.
```

Comment:
Ideally using a paste service, such as https://paste.gg.

Reply:
```diff
diff --git a/a b/b
index 124e49f..0de14c2 100644
--- a/a
+++ b/b
@@ -1,2 +1,5 @@
 **Stacktrace**
+
+```java
 Paste the stacktrace you get here.
+```
```

apperantly you need write access to use [comment suggestions](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/), silly

Reply:
I'm not (yet) convinced of the need to keep it off-github.

Reply:
especially since github supports details tags that can be used in templates to hide long snippets.